.gitignore

```diff
@@ -2,6 +2,7 @@
 build/
 */*/build
 */*/products
+*/*/sources
```

Comment:
```suggestion
```

Reply:
`L/Linux/sources/linuxkernel.config` is checked in, but maybe it is a good idea to gitignore `sources` anyway?